pkg/app/server/grpcapi/api.go

```diff
@@ -343,6 +344,46 @@ func (a *API) ListApplications(ctx context.Context, req *apiservice.ListApplicat
 	}, nil
 }
 
+func (a *API) UpdateApplication(ctx context.Context, req *apiservice.UpdateApplicationRequest) (*apiservice.UpdateApplicationResponse, error) {
```

Comment:
this change does not related to this pr, right? 👀

Reply:
Oops, I'll fix it 🙈